.github/workflows/linux-ppc64le.yml

```diff
@@ -0,0 +1,64 @@
+name: Native ppc64le Linux Test
+
+on:
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  native_ppc64le:
+    # To enable this workflow on a fork, comment out:
+    if: github.repository == 'numpy/numpy'
+    runs-on: ubuntu-24.04-ppc64le
+    
+    name: "Native PPC64LE"
+    steps:
+    
+    - uses: actions/checkout@v4
```

Comment:
Note that used actions should use a pinned hash, please see how it's done in other .yml files